### DISCUSSION TITLE
Marked as answer by (blank)

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#67009] [Mobile]
GitHub Mobile beta for Android 

- https://github.com/community/community/discussions/26311
![Screenshot_20230913-105235.png](https://github.com/community/community/assets/2119212/1fc37f6e-b9d6-4411-8e5c-400bf7e21991)

- Note that I have evidence this either has worked on the past or generally works: https://github.com/community/community/discussions/10817

**Comment by Akash1134** [comment]
Hi @jsoref ,

Nice catch! and thanks for sharing it. 

The "Marked answer by blank" is not mobile only -  on a desktop you will see it as "Answer Selected"

Reason is because they are all older discussions that were migrated from a different platform we were on. I looked at all the older comments that are [marked as answers](https://github.com/orgs/community/discussions?discussions_q=is%3Aopen+sort%3Atop+is%3Aanswered) and can confirm this happens for discussions older than June 2021(I don't have the exact timeframe right now). The missing GitHub user handle could be because the user does not exist anymore or that data was not migrated. I will reach out to the team to know the exact reason behind this.

**Reply by jsoref** [reply]
Thanks. Given that the poster still exists and that I'd expect the only people to be able to mark something as an answer are either repo/org owners or the poster, I'd imagine it's unlikely the marking account was deleted. (Thus, it seems more likely the data was not migrated/wasn't collected in the first place.)

Will you fix the mobile app to be more consistent with the desktop behavior, or are you just going to try to get all the data backfilled?

**Reply by jsoref** [reply]
@Akash1134 was there an update?

**Reply by jsoref** [reply]
Fwiw, it'd be easy enough to add some logic to change `marked as answer by ` to `marked as answer` if the marked by user field is empty...

**Comment by usman-302** [comment]
if you add another post you can add that as correct, even if its your own. just not the OP
